wiiu/include/sys/socket.h

```diff
@@ -22,12 +22,16 @@ extern "C" {
 
 #define SO_REUSEADDR    0x0004
 #define SO_NBIO         0x1014
+#define SO_NONBLOCK     0x1016
```

Comment:
Does the Wii U actually support non-blocking sockets? I feel like that's the sort of thing they'd leave out or relegate to SO* or whatever.

Reply:
Yep! It accepts the flag, and the read on the socket fails with EAGAIN as expected.